protocol/0043-MKTL-market_lifecycle.md

```diff
@@ -85,9 +85,9 @@ When a Market Proposal is not successful, see [governance proposal](./0028-GOVE-
 
 ### Pending
 
-When a Market Proposal is successful at the end of the voting period, the Market state becomes "Pending". Currently a Pending Market is always in an [auction call period](./0026-AUCT-auctions.md) that ends at the enactment date as specified in the Market Proposal. 
+When a Market Proposal is successful at the end of the voting period, the Market state becomes "Pending". Currently a Pending Market is always in an [auction call period](./0026-AUCT-auctions.md) that ends at the enactment date as specified in the Market Proposal if the other conditions for exiting auction period are met (liquidity committed, best static bid / ask present). If, initially,  at the specified enacment date it was not possible to leave the auction period, the auction will conclude as soon as possible once the conditions for auction exit are met. 
```

Comment:
Probably want to specify that a successor market will remain in pending until it's parent is settled, then (assuming other conditions are met) the opening auction will immediately resolve (in the same block I would expect?)

Reply:
I thought I'd want to allow to have the successor and parent running at the same time. For example CDS index markets do overlap... 

What you suggest would also make sense - would imply no overlap... so it's a decision. @barnabee thoughts?

Reply:
Running at the same time would also make sense, but for things like the insurance pool it would have to be shared for a while etc which might be a more complex setup than just shifting.

And CDS Indices intertwine terrifyingly indeed 😉.

Though that does bring up a thought. Assuming we do want market parent/successors overlapping in existence, do we want an ability to set said inheritance of liquidity/insurance pool to happen prior to settlement/termination? Settlement works fine for a rolling weekly etc but having that move occur at a more customisable point would allow futures to roll from on-the-run to off-the-run with liquidity etc moving to the new one whilst the old one still exists and settles at some later date.

Reply:
We allow overlaps.